blocks/header/header.js

```diff
@@ -52,6 +52,28 @@ function createMobileMenuControlsBlock() {
   return mobileMenuControls;
 }
 
+function createOverflowDropdown(navSections) {
+  const overflowDropdown = document.createElement('li');
+  overflowDropdown.classList.add('nav-button');
+  overflowDropdown.classList.add('nav-overflow');
```

Comment:
can be chained to `overflowDropdown.classList.add('nav-button', 'nav-overflow');`